shell/platform/embedder/embedder.cc

```diff
@@ -2306,13 +2306,16 @@ FlutterEngineResult FlutterEngineInitialize(size_t version,
         "Could not infer the Flutter project to run from given arguments.");
   }
 
+  // FIXME: This is probably the wrong place to call ConfigureShorebird, as
+  // some platforms (i.e., Windows) need to to swap out the app path before
+  // this point.
   // Begin shorebird
+#if FML_OS_MACOSX
```

Comment:
Is this only used on MacOS or also Android/iOS?

Reply:
Mac is the only platform we support that uses the embedder in this fashion. The windows engine calls ConfigureShorebird, so this is mac-only at the moment